docs/use-cases.rst

```diff
@@ -43,4 +43,155 @@ Typical Use-Cases
 
 .. START SEC:FAST-RTPS-IN-ROS2
 
+.. _fastrtps_ros2:
+
+Fast-RTPS in ROS 2
+==================
+
+Fast-RTPS is the default middleware implementation in the `Open Source Robotic Fundation (OSRF) <https://www.openrobotics.org/>`_
+`Robot Operating System ROS 2 <https://index.ros.org/doc/ros2/>`_. This tutorial is an explanation of how to take full
+advantage for Fast-RTPS wide set of capabilities in a ROS 2 project.
+
+The interface between the ROS2 stack and Fast-RTPS is provided by a ROS 2 package
+`rmw_fastrtps <https://raw.githubusercontent.com/ros2/rmw_fastrtps/>`_. This packages is available in all ROS 2
+distributions, both from binaries and from sources. Event thought Fast-RTPS is the default ROS 2 middleware, it is still
+possible to explicitly specify it in two different ways:
```

Comment:
I would remove this sentence: "Event thought Fast-RTPS is the default ROS 2 middleware, it is still
possible to explicitly specify it in two different ways:" and move the 2 possibilities at the end.

Reply:
See the comment below.

Reply:
done in fb916ea